wallet/chainntfns.go

```diff
@@ -293,6 +294,15 @@ func (w *Wallet) onBlockConnected(serializedBlockHeader []byte, transactions [][
 	w.NtfnServer.notifyMainChainTipChanged(chainTipChanges)
 	w.NtfnServer.sendAttachedBlockNotification()
 
+	// Parse vote version from extended bits.  Note that once ExtendedBits
+	// are used for other information as well this will need to be updated
+	// to accomodate.
+	voteVersion := binary.LittleEndian.Uint32(w.VoteBits().ExtendedBits)
```

Comment:
This should only pull out the version if the extended bits are long enough.

Also need to double check that the version is at the beginning of the bytes rather than the end.  In wallet's case it only is including the version in the extended bits so I'm not sure which way is correct to pull them out.  Need to consult dcrd here.